core/templates/pages/admin-page/misc-tab/admin-misc-tab.directive.ts

```diff
@@ -155,6 +156,19 @@ angular.module('oppia').directive('adminMiscTab', [
             });
         };
 
+        ctrl.retrieveExplorationInteractionIds = function() {
+          $http.get(INTERACTIONS_BY_ID_HANDLER_URL, {
```

Comment:
Add a statusMessage `Retrieving interactions in exploration ...`

Reply:
done